src/templates/components/CommunityTemplateReadme.tsx

```diff
@@ -34,9 +31,6 @@ class CommunityTemplateReadmeUnconnected extends Component<Props> {
     return (
       <MarkdownRenderer
         text={readme}
-        cloudRenderers={{
-          image: cloudImageRenderer,
-        }}
```

Comment:
is this the bug? what was the main issue?

Reply:
The main issue was that we needed to add the flag `escapeHtml={false}` to ReactMarkdown component